test/transaction-runner.ts

```diff
@@ -124,7 +124,7 @@ describe('TransactionRunner', () => {
     describe('getNextDelay', () => {
       const FAKE_RETRY_INFO = Buffer.from('fake-retry-info');
 
-      const FAKE_ERROR: ServiceError = new Error('err');
+      const FAKE_ERROR = new Error('err') as ServiceError;
```

Comment:
Just curious, why do we prefer this form of casting? Just want to know the reasoning so I can avoid doing it wrong in the future. 😎

Reply:
Oh, my bad, for some reason I didn't think `code` was a required property, that makes sense then.